salt/states/junos.py

```diff
@@ -337,7 +371,7 @@ def install_config(name, **kwargs):
 
     """
     ret = {"name": name, "changes": {}, "result": True, "comment": ""}
-    ret["changes"] = __salt__["junos.install_config"](name, **kwargs)
+    ret["changes"] = __salt__["junos.install_config"](path, **kwargs)
```

Comment:
Above you had name, path, kwargs, and now it is just a swap of name for path, see comment above

Reply:
reverted via https://github.com/saltstack/salt/pull/57281/commits/682cb0a08baa14e1fa4684163c5588cbda672d25#diff-442286d51776ddefc791bc7f15e42fc2R352